utils/build/webpack.base.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const packageManifest = require(path.resolve('package.json'));
+
+const options = {
+  mode: 'production',
+  entry: path.resolve('src', 'index.js'),
+  devtool: 'source-map',
```

Comment:
Do we distribute the sourcemap?

Reply:
With this setup yes. I don't see any downsides to providing it for the commonjs and UMD bundle. Do you?